gutter.go

```diff
@@ -76,18 +80,28 @@ func glfwCursorPositionCallbackAtPhase(
 
 func glfwMouseButtonCallback(window *glfw.Window, key glfw.MouseButton, action glfw.Action, mods glfw.ModifierKey) {
 
-	if key == glfw.MouseButton1 && action == glfw.Press {
+	if key == glfw.MouseButton1 {
 		x, y := window.GetCursorPos()
-		glfwCursorPositionCallbackAtPhase(window, flutter.KDown, x, y)
-		window.SetCursorPosCallback(func(window *glfw.Window, x float64, y float64) {
-			glfwCursorPositionCallbackAtPhase(window, flutter.KMove, x, y)
-		})
-	}
 
-	if key == glfw.MouseButton1 && action == glfw.Release {
-		x, y := window.GetCursorPos()
-		glfwCursorPositionCallbackAtPhase(window, flutter.KUp, x, y)
-		window.SetCursorPosCallback(nil)
+		// recalculate x and y from screen cordinates to pixels
+		widthPx, _ := window.GetFramebufferSize()
+		width, _ := window.GetSize()
+		pixelsPerScreenCoordinate := float64(widthPx) / float64(width)
+		x = x * pixelsPerScreenCoordinate
+		y = y * pixelsPerScreenCoordinate
```

Comment:
GetCursorPos returns the cursor position in screen coordinates, but flutter expects the click location to be in pixels.
On most machines the pixel to screen coordinates ratio is 1.0, which is why this bug has probably gone unnoticed so far. This calculation fixes click position for users who have pixelsPerScreenCoordinate != 1.0

Reply:
Thanks for the clarification, It was needed :+1: